contrib/alarm/alarm.cpp

```diff
@@ -0,0 +1,88 @@
+/* FILE: alarm.cpp
+ * AUTHOR: Rohan Garg
+ * EMAIL: rohgarg@ccs.neu.edu
+ * Copyright (C) 2015 Rohan Garg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+
+#include "dmtcp.h"
+#include "jassert.h"
+
+#ifdef ALARM_PLUGIN_DEBUG
+# undef JTRACE
+# define JTRACE JNOTE
+#endif
```

Comment:
Instead of this block here, you could just do `#define DEBUG` before including jassert.h. That would allow you to see JTRACE output on console (there won't be any jassertlogs anyways).